### PR TITLE
fix: username-flag alignment fixed

### PR DESCRIPTION
**Pull request by Abhikumar98**
## Issue

https://github.com/subeshb1/developer-community-stats/issues/240

## Description

Screenshot 2020-10-18 at 10 38 33 AM

## Checklist

- [x] Added contributor object in ascending order in `contributors.json`
- [x] Github action build passing
- [x] No merge conflicts

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -2,25 +2,22 @@ const path = require(`path`)
 const { createFilePath } = require(`gatsby-source-filesystem`)
 const contributorsList = require('./contributors.json')
 let stats
-if (process.env.ENV_TYPE === 'mock') {
+if (true) {
```
Comment (subeshb1) [inline gatsby-node.js:5]: Let's revert the changes in this file

Comment (Abhikumar98) [reply gatsby-node.js:5]: yes, doing it rn.

Comment (Abhikumar98) [reply gatsby-node.js]: @Shubham-Agarwal16 Changes for the file reverted.

**Comment by subeshb1** [comment]
It looks like adding ant-design has overridden the styles of the website.

Before:
Screen Shot 2020-10-18 at 11 02 39

After:
Screen Shot 2020-10-18 at 11 02 29

The alignments have also broken in navbar and font size changes in the footer


Do you know a way to prevent ant-design to override global styles?

**Comment by Abhikumar98** [comment]
> @Abhikumar98
> 
> It looks like adding ant-design has overridden the styles of the website.
> 
> Before:
> Screen Shot 2020-10-18 at 11 02 39
> 
> After:
> Screen Shot 2020-10-18 at 11 02 29
> 
> The alignments have also broken in navbar and font size changes in the footer
> 
> Do you know a way to prevent ant-design to override global styles?

Not sure about how to prevent the styles from being overridden, but fixed the styling by overriding with custom CSS.

**Comment by subeshb1** [comment]
I don't think it will be feasible to add CSS everywhere. Ant-design has overridden a lot of styles. We will have to look into alternatives. Plus it adds in a lot of sizes to CSS bunfle

**Comment by Abhikumar98** [comment]
> I don't think it will be feasible to add CSS everywhere. Ant-design has overridden a lot of styles. We will have to look into alternatives. Plus it adds in a lot of sizes to CSS bunfle

Yes, makes sense

**Comment by subeshb1** [comment]
Closing this
